tests/unittests/ExecutorTest.cpp

```diff
@@ -786,9 +785,9 @@ TEST_F(ThreadPoolExecutorTest, MultiNode) {
   // the ThreadPoolExecutor has a reference to) and the TestDeviceManager map
   // (which the ExecutorTestBuilder has a reference to).
   auto deviceManager =
-      std::make_shared<TestDeviceManager>(deviceManagerThreads);
-  deviceManagerMap_.insert(std::make_pair(testDeviceId, deviceManager));
-  testDeviceManagerMap_.insert(std::make_pair(testDeviceId, deviceManager));
+      llvm::make_unique<TestDeviceManager>(deviceManagerThreads);
+  deviceManagerMap_.insert(
```

Comment:
nit: deviceManagerMap_.emplace(testDeviceId, std::move(deviceManager)) and in other places.

Reply:
@rdzhabarov I didn't write that but yeah I agree it's better I'll change it